docs/changes/utils/13730.bugfix.rst

```diff
@@ -0,0 +1 @@
+Update ``_rmtree`` to catch ``OSError`` and use ``shutil.move`` if errno == EXDEV.
```

Comment:
Since `_rmtree` is "hidden", it should not be explicitly mentioned like this in a change log. But rather, what do you want the end-user to know about this patch? How is the `data.py` behavior changing for them at a high level?